service/controller/cluster.go

```diff
@@ -30,10 +30,11 @@ import (
 )
 
 type ClusterConfig struct {
-	G8sClient    versioned.Interface
-	K8sClient    kubernetes.Interface
-	K8sExtClient apiextensionsclient.Interface
-	Logger       micrologger.Logger
+	CertsSearcher certs.Interface
```

Comment:
There is some refactoring because of the injection of the certs searcher. This is because different controllers need it.